Source/Glide64/Config.cpp

```diff
@@ -1310,3 +1310,28 @@ void CALL DllAbout ( HWND hParent )
   hostWindow = NULL;
 #endif
 }
+
+void general_setting(short setting_ID, const char * name, unsigned int value)
+{
+    RegisterSetting(
+        setting_ID,
+        Data_DWORD_General,
+        name,
+        NULL,
+        value,
+        NULL
+    );
+    return;

```

Comment:
AFAIK every function has a return point.
The way I understand it, the `return` keyword is optional only in void functions that don't return an actual thing, but still need to leave the function.

I guess it depends on whether you view C as a low-level or a high-level language, but personally I think it's better to be explicit and formalize when you know it's time to be done with the function.  Although, I can't say it's any more or less maintainable to just remove the `return;`, so I think it's fine if you want to remove that.

I see it more as a matter of the C language permitting the omission of the `return;` in void functions, rather than creating it as an optional thing just if you wanted it, but I guess I could be wrong on that.